Rendering/render.h

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+
+
+
+class Render {
+
+public:
+
+	void initialize() {
+		std::cout << "Render init" << std::endl;
+	}
+
+	void render() {
```

Comment:
Inconsistent function naming scheme

Reply:
What name would you suggest?

Reply:
Let's follow Google C++ standards 

https://google.github.io/styleguide/cppguide.html#Function_Names